willy/Week6/cv-fullstack/frontend/src/app/cv/cv.component.html

```diff
@@ -0,0 +1,37 @@
+<main class="main__container">
+    <div class="container__izq">
```

Comment:
Don´t mix English and Spanish for classes names

Reply:
y no hacer abreviaciones